### ISSUE TITLE
License of OpenAssistant/oasst-sft-4-pythia-12b-epoch-3.5

### ISSUE THREAD

**Issue by debraj135** [LAION-AI/Open-Assistant#2861]
I would like to understand whether the apache 2.0 license on [this](https://huggingface.co/OpenAssistant/oasst-sft-4-pythia-12b-epoch-3.5) particular model card is appropriate.

I noticed a related discussion about this [here](https://huggingface.co/OpenAssistant/oasst-sft-4-pythia-12b-epoch-3.5/discussions/1).

**Comment by olliestanley** [comment]
Yes, the model is licensed under Apache 2.0 and this will not be changing

**Comment by andreaskoepf** [comment]
@debraj135 basically all currently existing LLMs are trained with parts of web-scrapes like Common-Crawl (CC) .. none of the groups who trained them have fully licensed the whole internet nor have they really checked that all data is public domain or copyright free (it is obvious that it is not). Whether training a neural network on data is an copyright infringement or not will be determined in the next round of legislation. Adjusting weights during a gradient descent step is VERY different from directly copying or redistributing data, it is more like learning. Lawmakers will have to explain why it is OK for a human to read a book but not for a robot.

**Comment by debraj135** [comment]
@olliestanley @andreaskoepf Thank you for your prompt response!
